packages/ckeditor5-source-editing/src/sourceediting.ts

```diff
@@ -228,12 +230,12 @@ export default class SourceEditing extends Plugin {
 			const domSourceEditingElementTextarea = createElement( domRootElement.ownerDocument, 'textarea', {
 				rows: '1',
 				'aria-label': 'Source code editing area'
-			} );
+			} ) as HTMLTextAreaElement;
```

Comment:
This won't be needed if merged after https://github.com/ckeditor/ckeditor5/pull/13386/files.